### PR TITLE
feat(ca-metrics): remove old code for CA

### PR DESCRIPTION
**Pull request by torgeadelin**
# COMPLETES #[SPARK-440534](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-440534)

## This pull request addresses

-remove old CA related code

## by making the following changes

- deleting code

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [x] Internal code refactor

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/src/metrics/index.ts [hunk 1]
```diff
@@ -83,27 +19,6 @@ class Metrics {
    */
   constructor() {
     if (!Metrics.instance) {
```
Comment (marcin-bazyl) [inline packages/@webex/plugin-meetings/src/metrics/index.ts:21]: hmmm this old code is effectively a singleton - how did this work without hitting the same issue you had?

**Review by marcin-bazyl — APPROVED** [review]

**Comment by robstax** [comment]
image
😍

**Comment by csshz** [comment]
Catalin, If this pr is uploaded, does it mean that the SDK loses the ability to send DiagnosticEvent, whether the new sending interface is already working, and how can we ensure that the sent CA data will not be lost.

File: packages/@webex/plugin-meetings/src/metrics/index.ts (hunk 1 above)
Comment (torgeadelin) [reply packages/@webex/plugin-meetings/src/metrics/index.ts:21]: We suspect because it was in the same module, and not in a different plugin

**Comment by torgeadelin** [comment]
> Catalin, If this pr is uploaded, does it mean that the SDK loses the ability to send DiagnosticEvent, whether the new sending interface is already working, and how can we ensure that the sent CA data will not be lost at present.

Hi, this PR only removes the old code (which is no longer used in this branch). The new code is already in, including submitting the client events for CA.

**Pull request merged**